templates/human.html

```diff
@@ -210,18 +210,18 @@ <h4>Human behavior configuration:</h4>
         extra = renderExtra(extra, platform);
         switch (platform) {
             case "darwin":
-                baseHuman = 'curl -sk -o '+humanName+'.tar.gz -X POST -H \'file:'+humanName+'.tar.gz\' '+http+'/file/download 2>&1 && mkdir '+humanName+' && tar -C '+humanName+' -zxvf '+humanName+'.tar.gz ' +
-                    '&& cd '+humanName+' && virtualenv -p python3 '+humanName+' && '+humanName+'/bin/pip install -r requirements.txt && '+humanName+'/bin/python human.py --clustersize '+taskCount+' ' +
+                baseHuman = 'curl -sk -o \''+humanName+'.tar.gz\' -X POST -H \'file:'+humanName+'.tar.gz\' '+http+'/file/download 2>&1 && mkdir \''+humanName+'\' && tar -C \''+humanName+'\' -zxvf \''+humanName+'.tar.gz\' ' +
+                    '&& cd \''+humanName+'\' && virtualenv -p python3 \''+humanName+'\' && \''+humanName+'/bin/pip\' install -r requirements.txt && \''+humanName+'/bin/python\' human.py --clustersize '+taskCount+' ' +
                     '--taskinterval '+taskInterval+' --taskgroupinterval '+taskClusterInterval+' --extra '+extra;
                 break;
             case "linux":
-                baseHuman = 'curl -sk -o '+humanName+'.tar.gz -X POST -H \'file:'+humanName+'.tar.gz\' '+http+'/file/download 2>&1 && mkdir '+humanName+' && tar -C '+humanName+' -zxvf '+humanName+'.tar.gz ' +
-                    '&& cd '+humanName+' && virtualenv -p python3 '+humanName+' && '+humanName+'/bin/pip install -r requirements.txt && '+humanName+'/bin/python human.py --clustersize '+taskCount+' ' +
+                baseHuman = 'curl -sk -o \''+humanName+'.tar.gz\' -X POST -H \'file:'+humanName+'.tar.gz\' '+http+'/file/download 2>&1 && mkdir \''+humanName+'\' && tar -C \''+humanName+'\' -zxvf \''+humanName+'.tar.gz\' ' +
+                    '&& cd \''+humanName+'\' && virtualenv -p python3 \''+humanName+'\' && \''+humanName+'/bin/pip\' install -r requirements.txt && \''+humanName+'/bin/python\' human.py --clustersize '+taskCount+' ' +
                     '--taskinterval '+taskInterval+' --taskgroupinterval '+taskClusterInterval+' --extra '+extra;
                 break;
             case "windows-psh":
                 baseHuman = '$server="'+http+'"; $url="$server/file/download"; $wc=New-Object System.Net.WebClient; $wc.Headers.add("file","'+humanName+'.zip"); $wc.DownloadFile($url, "$pwd\\'+humanName+'.zip"); ' +
-                    'Expand-Archive '+humanName+'.zip -DestinationPath '+humanName+' -Force; cd '+humanName+'; python.exe -m venv '+humanName+';' +
+                    'Expand-Archive "'+humanName+'.zip" -DestinationPath "'+humanName+'" -Force; cd "'+humanName+'"; python.exe -m venv "'+humanName+'";' +
                     '.\\'+humanName+'\\Scripts\\pip.exe install -r requirements.txt; .\\'+humanName+'\\Scripts\\python.exe human.py --clustersize '+taskCount+' ' +
```

Comment:
The spaces here seems to still be causing issues for me (tested with a human named "My Human"). I'm thinking since this is windows-psh, you could use the following instead for both running `pip` and `python human.py`: `Start-Process -FilePath ".\\My Human\\Scripts\\pip.exe" -ArgumentList "install -r requirements.txt"; Start-Process -FilePath ".\\My Human\\Scripts\\python.exe" -ArgumentList "human.py --clustersize..."` which should resolve the spacing issues.